packages/tunnel/src/commands/deploy/utils.ts

```diff
@@ -0,0 +1,161 @@
+import { appendPath } from '@silverhand/essentials';
+import AdmZip from 'adm-zip';
+import chalk from 'chalk';
+import ora from 'ora';
+
+import { consoleLog } from '../../utils.js';
+
+type TokenResponse = {
+  access_token: string;
+  token_type: string;
+  expires_in: number;
+  scope: string;
+};
+
+type DeployArgs = {
+  auth: string;
+  endpoint: string;
+  experiencePath: string;
+  managementApiResource?: string;
+  verbose: boolean;
+};
+
+export const deployToLogtoCloud = async ({
+  auth,
+  endpoint,
+  experiencePath,
+  managementApiResource,
+  verbose,
+}: DeployArgs) => {
+  const spinner = ora();
```

Comment:
what if it's non-verbose? will it start spinning?

Reply:
Yes, there's another more generic spinning message for non-verbose mode:

<img width="458" alt="image" src="https://github.com/user-attachments/assets/1c1f36f2-25d0-446e-ae6c-0fa75092df7f">
<img width="421" alt="image" src="https://github.com/user-attachments/assets/6432bbbb-b2dd-4e15-822e-028436efb307">